ligo/stablecoin/actions.ligo

```diff
@@ -39,13 +39,13 @@
 function authorize_pending_owner
   ( const store : storage
   ; const full_param : closed_parameter
-  ) : storage * address is case store.roles.pending_owner of
-    Some (pending_owner) ->
+  ) : storage * address is case store.roles.pending_owner of [
```

Comment:
Is the `[  ]` around  patterns a new addition to LIGO ?

Reply:
Yup, it's required now, it replaces the `end` keyword.